packages/zowe-explorer/package.nls.json

```diff
@@ -59,8 +59,8 @@
   "refreshAllJobs": "Refresh Jobs View",
   "addJobsSession": "Add Profile to Jobs View",
   "removeJobsSession": "Hide Profile",
-  "downloadSpool": "Download Spool",
-  "downloadSpoolBinary": "Download Spool (Binary)",
+  "downloadSpool": "Download All Spool",
+  "downloadSpoolBinary": "Download All Spool (Binary)",
```

Comment:
Thoughts on using "Download All" here? Since the single spool options were designated as "Download," we could match that same naming convention